### DISCUSSION TITLE
FIT image?

### DISCUSSION THREAD

**Discussion by mbilloo** [OE4T/meta-tegra#725] [Q&A] [answered]
Hi,

Has there been success with getting a FIT image to load in u-boot? I'm going down a rabbit hole where u-boot is stuck uncompressing the kernel and I have a feeling there's an overflow issue in the inflation of the gzip image (I'm stepping through the inflate_fast function). Just wanted to see if you had any success.

Thanks

**Comment by ichergui** [comment, marked as answer]
Hey @mbilloo 

`FIT image` is supported for the following Tegra paltform: 
- `Jetson Nano (SDcard version)` 
- `Jetson Nano (eMMC version)`
- `Jetson TX2`

Please refer to the [wiki page](https://github.com/OE4T/tegra-demo-distro/wiki/Tegra-&-Uboot-Flattened-Image-Tree-(FIT-image))
